packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -263,7 +263,7 @@ const New = decorate([
             [id]: DEFAULT_SCHEDULE,
           }
           settings = {
-            '': state.settings && state.settings.get(''),
+            '': state.settings && state.settings?.get(''),
```

Comment:
`state.settings && ...` already checks for falsy values, so the `?.` is redundant here.
See the `state.settings && ...` like an `if` condition.
```js
if(state.settings){
  settings[''] = state.settings.get('')
}
```
[Logical AND operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND)